libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -161,6 +161,18 @@ export interface AccountBridge<T extends TransactionCommon> {
     parentAccount?: Account | null | undefined;
     transaction?: T | null | undefined;
   }): Promise<BigNumber>;
+  // reconciliation by the React definition. https://reactjs.org/docs/reconciliation.html
+  applyReconciliation?: (
+    account: Account,
+    updatedRaw: AccountRaw,
+    next: Account
+  ) => boolean;
+  // convert account to accountRaw
+  assignToAccountRaw?: (account: Account, accountRaw: AccountRaw) => void;
+  // convert accountRaw to account
+  assignFromAccountRaw?: (accountRaw: AccountRaw, account: Account) => void;
+  // make a initial account
+  initAccount?: (account: Account) => void;
```

Comment:
Here is a suggested improvement in documented these APIs in the js comment

```js
/**
 * This function applies the change from moving from "account" to an updated version ("updatedRow").
 * It returns a boolean value indicating if something changed in the "next" Account that possibly gets mutated in-place.
 *
 * @param {Account} account - The original account object.
 * @param {AccountRaw} updatedRaw - The updated version of the account in its serialized form.
 * @param {Account} next - The updated account object.
 * @returns {boolean} - Indicates if something changed in the "next" account.
 */
   applyReconciliation?: (
    account: Account,
    updatedRaw: AccountRaw,
    next: Account
  ) => boolean;


/**
 * This function mutates the 'accountRaw' object in-place to add any extra fields that the coin may need to set.
 * It is called during the serialization mechanism, for instance bitcoinResources need to be serialized.
 *
 * @param {Account} account - The original account object.
 * @param {AccountRaw} accountRaw - The account in its serialized form.
 */
  assignToAccountRaw?: (account: Account, accountRaw: AccountRaw) => void;

/**
 * This function mutates the 'account' object in-place to add any extra fields that the coin may need to set.
 * It is called during the deserialization mechanism, for instance bitcoinResources need to be deserialized.
 *
 * @param {AccountRaw} accountRaw - The account in its serialized form.
 * @param {Account} account - The original account object.
 */
  assignFromAccountRaw?: (accountRaw: AccountRaw, account: Account) => void;
/**
 * This function mutates the 'account' object to extend it with any extra fields of the coin.
 * For instance bitcoinResources need to be created.
 *
 * @param {Account} account - The original account object to mutates in-place.
 */
  initAccount?: (account: Account) => void;
```

Reply:
Done thx